docs/team/kaoxi998533.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Hi there! I'm Eddy Fan! 😁  
+---
+
+<img src="../images/kaoxi998533.png" alt="Eddy Fan's picture" width="200px" style="display: block; margin: 0 auto" />
```

Comment:
To prevent errors coming from using local paths, the code should use the image link created: https://ay2425s1-cs2103t-w14-4.github.io/tp/images/kaoxi998533.png

Reply:
Yep, do use the image link!